include/io.h

```diff
@@ -0,0 +1,149 @@
+#ifndef MPM_IO_H_
+#define MPM_IO_H_
+
+#include <fstream>
+#include <iostream>
+#include <memory>
+#include <sstream>
+#include <utility>
+
+#include <boost/filesystem.hpp>
+#include <eigen3/Eigen/Dense>
+
+//! Alias for JSON
+#include "json.hpp"
+using json = nlohmann::json;
+
+//! \brief Input/Output handler
+class IO {
+ public:
+  //! Constructor
+  IO(json json_filename);
+
+  //! Return input mesh file name
+  std::string inputMeshFileName() const { return inputMeshFileName_; }
+
+  //! Return input sub mesh file name
+  std::string inputSubMeshFileName() const { return inputSubMeshFileName_; }
```

Comment:
Can we store the file names in a map and return the value for each key instead of multiple file names.